driver/driver_config_builder.py

```diff
@@ -296,9 +296,8 @@ def from_env_vars(self):
 
         if self.config["db_type"] == "postgres":
             if not db_name:
-                msg = ("Must supply database name for Postgres via environment variable: "
-                       "POSTGRES_OTTERTUNE_DB_NAME")
-                raise DriverConfigException(msg)
+                # Default to postgres
+                db_name = "postgres"
```

Comment:
Let's not set the default name here. I think we already set the default PG name in the cloudformation format. 
If the dbname is empty, we still should throw exceptions I think

Reply:
One reason I wanted to do it hear is to make it work for all setups (docker, manual, etc..) I think it makes sense to have a default in the agent, just like we have a default for other things like collection intervals. I think it would be inconsistent to only have a default for cloudformation and not the other deployment methods.

Reply:
I think it makes sense to have a default set for the agent. Fewer things can go wrong as @michaelhermenault mentioned for multiple deployment methods. We have had a few users want to run it directly from source code. We can make it clear in agent documentation that users can override the database name using the environment variable.

However, I don't think this is the place to set the default. I think it is better to have the default set here - https://github.com/ottertune/ot-agent/blob/main/driver/driver_config_builder.py#L291. 

```lang=python
postgres_db_name = os.getenv("POSTGRES_OTTERTUNE_DB_NAME", "postgres")
```